CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# leto-modelizer-ia-api
+
+## Description
+This file describe how to contribute and develop to this open source projet.
```

Comment:
This file describe**s** how to contribute to and develop for this open-source project.